### DISCUSSION TITLE
Create listings in bulk via your API or other non-manual means?

### DISCUSSION THREAD

**Discussion by nhtahoe** [Sellbrite/Sellbrite-API#8] [Q&A]
Do you have anything on your roadmap that would let me create listings in bulk via your API or other non-manual means? E.g. Datafeedwatch has a feature where I can specify a HTTP feed location and it will daily check for changes and auto-create new listings based that: https://help.datafeedwatch.com/article/q0m5q4tnb2-3202690-uploading-data-feeds-via-http.

**Comment by rengler-godaddy** [comment]
Hello - Thanks for reaching out! 

We don’t have anything currently on our roadmap that would allow you to build listings in bulk through our API. Thanks for providing that datafeed link. I will pass this feedback to our product team and maybe this is something that we can look into building out later down the line.
